src/index.html

```diff
@@ -9,9 +9,182 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <div>
+          <label>
+              Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              maxlength="124"
+            >
+          </label>
+        </div>
+
+        <div class="form_field">
+          <label>
+              Name:
+            <input
```

Comment:
Go through [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix your code style 
```suggestion
          <label>
            Surname:
            <input
              type="text"
              name="surname"
              autocomplete="off"
              maxlength="124"
            >
          </label>
        </div>

        <div class="form_field">
          <label>
            Name:
            <input
```